secondarylib/src/main/scala/com/socrata/datacoordinator/secondary/SecondaryWatcher.scala

```diff
@@ -175,24 +182,82 @@ class SecondaryWatcher[CT, CV](universe: => Managed[SecondaryWatcher.UniverseTyp
   }
 }
 
+object SecondaryWatcherClaimManager {
+  private val log = LoggerFactory.getLogger(classOf[SecondaryWatcherClaimManager])
+
+  // in-memory list of datasets that are actively being worked on by _this_ instance
+  private val whitelist = scala.collection.mutable.Set.empty[Long]
+
+  def whitelistAdd(datasetId: DatasetId): Unit = {
+    if (whitelist.contains(datasetId.underlying)) {
+      log.error("We have already claimed dataset {} in our whitelist. An unexpected error has occurred; exiting.", datasetId)
```

Comment:
I don't believe that we do. (no usages in repo and `value fatal is not a member of org.slf4j.Logger`)